aspnetcore/tutorials/razor-pages/page.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Scaffolded Razor Pages in ASP.NET Core
+author: rick-anderson
+description: Explains the Razor Pages generated by scaffolding.
+keywords: ASP.NET Core,Razor Pages,Razor,MVC
+ms.author: riande
+manager: wpickett
+ms.date: 7/27/2017
+ms.topic: get-started-article
+ms.technology: aspnet
+ms.prod: aspnet-core
+uid: tutorials/razor-pages/page
+---
+# Scaffolded Razor Pages in ASP.NET Core
+
+By [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+This tutorial examines the Razor Pages created by scaffolding in the [previous tutorial](xref:tutorials/razor-pages/page). 
+
+[View or download](https://github.com/aspnet/Docs/tree/master/aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie) sample.
+
+## The Create, Delete, Details, and Edit pages.
+
+Examine the *Pages/Movie/Index.cshtml.cs* code-behind file:
+
+[!code-csharp[Main](razor-pages-start/snapshot_sample/RazorPagesMovie/Pages/Movie/Index.cshtml.cs)]
+
+Razor Pages are derived from `PageModel`. By convention, the `PageModel` derived class is called `<PageName>Model`. The constructor uses [dependency injection](xref:fundamentals/dependency-injection) to add the `MovieContext` to the page. All the scaffolded pages follow this pattern.
+
+When a request is made for the page, the `OnGetAsync` method returns a list of movies to the Razor Page. `OnGetAsync` or `OnGet` is called on a Razor Page to initialize the state for the page. In this case, `OnGetAsync` gets a list of movies to display.
+
+Examine the *Pages/Movie/Index.cshtml* Razor page:
+
+[!code-cshtml[Main](razor-pages-start/snapshot_sample/RazorPagesMovie/Pages/Movie/Index.cshtml)]
+
+Razor can transition from HTML into C# or into Razor specific markup. When an `@` symbol is followed by a [Razor reserved keyword](xref:mvc/views/razor#razor-reserved-keywords) it transitions into Razor specific markup, otherwise it transitions into C#.
+
+The `@page` Razor directive makes the file into an MVC action - which means that it can handle requests. `@page` must be the first Razor directive on a page. `@page` is an example of transitioning into Razor specific markup.  See [Razor syntax](xref:mvc/views/razor#razor-syntax) for more information.
```

Comment:
- Use an mdash instead of a dash before "which means"
- Hyphenate "Razor specific"
- Remove extra space before "See"